internal/testgen/gen_encoded_compare.go

```diff
@@ -273,6 +273,19 @@ images:
 			{Fn: "YAMLEq", Argsf: `"kind: Kustomization", "kind: Kustomization"`}, // Not supported.
 			{Fn: "YAMLEq", Argsf: "raw, conf"},
 			{Fn: "YAMLEq", Argsf: "raw, string(respBytes)"},
+
+			// these are meaningless, they should be ignored.
+			{Fn: "Equal", Argsf: "42, resultJSON"},
+			{Fn: "EqualValues", Argsf: "42, resultJSON"},
+			{Fn: "Exactly", Argsf: "42, resultJSON"},
+			{Fn: "Equal", Argsf: "42, conf"},
+			{Fn: "EqualValues", Argsf: "42, conf"},
+			{Fn: "Exactly", Argsf: "42, conf"},
+			{Fn: "Equal", Argsf: "`{\"foo\":\"bar\"}`, respJSONRawMessage"},
```

Comment:
It should not be ignored, this is analogue of cases above with `resultJSONBytes` / `respBytes`